modules/wolfgame.py

```diff
@@ -4120,13 +3999,13 @@ def investigate(cli, nick, chan, rest):
         vrole = var.FINAL_ROLES[victim]
     pm(cli, nick, ("The results of your investigation have returned. \u0002{0}\u0002"+
                    " is a... \u0002{1}\u0002!").format(victim, vrole))
-    var.LOGGER.logBare(victim, "INVESTIGATED", nick)
+    debuglog("{0} ({1}) ID: {2} ({3})".format(nick, var.get_role(nick), victim, vrole))
     if random.random() < var.DETECTIVE_REVEALED_CHANCE:  # a 2/5 chance (should be changeable in settings)
         # The detective's identity is compromised!
         for badguy in var.list_players(var.WOLFCHAT_ROLES):
             pm(cli, badguy, ("Someone accidentally drops a paper. The paper reveals "+
                             "that \u0002{0}\u0002 is the detective!").format(nick))
-        var.LOGGER.logBare(nick, "PAPERDROP")
+        debuglog("{0} ({1}) PAPERDROP: {2}".format(nick, var.get_role(nick), " ".join(var.list_players(var.WOLFCHAT_ROLES))))
```

Comment:
Listing wolves probably isn't necessary. I can do the PAPER DROP instead of PAPERDROP, I'll wait until it's reviewed a bit more before pushing that one though.